docsets/Vaadin/docset.json

```diff
@@ -11,6 +11,10 @@
         {
             "_comment": ""
         },
+        {
+            "version": "8.25.1",
+            "archive": "versions/8.25.1/Vaadin.tgz"
```

Comment:
This file doesn't exist. It was added at `docsets/Vaadin/versions/Vaadin.tgz` instead I think.